src/app/seeyond/dimensions/dimensions.component.ts

```diff
@@ -64,15 +64,24 @@ export class DimensionsComponent implements OnInit {
         break;
 
       case "radius":
-        if(measurement < (this.feature.width *.5) + 1) {
-          this.feature.radius = (this.feature.width *.5) + 1;
-          this.alert.error('The radius must be greater than half the width. Radius set to: ' + this.feature.radius);
+        const halfRadius = (this.feature.width *.5) + 1;
+        if(measurement < 30) {
+          if (halfRadius < 30 ) {
```

Comment:
Clean this up a bit. An idea would be to make this `if ( measurement < 30 && halfRadius < 30)` for example.